Update bridge tutorial (#513)
OffchainLabs/arbitrum-docs: arbitrum-docs/getting-started-users.mdx

The only prerequisite for this quickstart is to have a web3 wallet installed, e.g. Metamask. If you don't have one, visit Arbitrum's [portal page](https://portal.arbitrum.io/?categories=wallet) to download one.

## Deposit ETH or ERC-20 tokens (from parent chain to child chain)

### Step 1: Get some native currency

You'll need the native currency of the parent chain to be able to bridge your assets from it to the destination chain. For example, if you want to bridge assets from Ethereum to Arbitrum One, you'll need ETH on Ethereum to initiate the process.

There are several ways to obtain the native currency:

- Using a [supported centralized exchange](https://portal.arbitrum.io/?categories=centralized-exchanges), which allows you to purchase ETH and withdraw it to your wallet.
- Using an [on-ramp service](https://portal.arbitrum.io/?categories=fiat-on-ramp), which allows you to purchase ETH and send it directly to your wallet.

> - If you are using a testnet, requesting funds from a faucet for [Sepolia](https://sepoliafaucet.com/), [Goerli](https://goerlifaucet.com) or [Arbitrum Sepolia](https://faucet.quicknode.com/arbitrum/sepolia)

Reviewer: ```suggestion
- If you are using a testnet, requesting funds from a faucet for [Sepolia](https://sepoliafaucet.com/), [Goerli](https://goerlifaucet.com) or [Arbitrum Sepolia](https://faucet.quicknode.com/arbitrum/sepolia).
```

> ### Step 2: Add the network to your wallet

Reviewer: should we say "Add the preferred network to your wallet"?

Here we display the information of the most common Arbitrum chains, but you can find a more exhaustive list in our [RPC endpoints and providers](/node-running/node-providers.mdx) page.

| Parameter          | Arbitrum One                 | Arbitrum Nova                | Arbitrum Sepolia (testnet)             |
| ------------------ | ---------------------------- | ---------------------------- | -------------------------------------- |
| Network name       | Arbitrum One                 | Arbitrum Nova                | Arbitrum Sepolia                       |
| RPC URL            | https://arb1.arbitrum.io/rpc | https://nova.arbitrum.io/rpc | https://sepolia-rollup.arbitrum.io/rpc |
| Chain ID           | 42161                        | 42170                        | 421614                                 |
| Currency symbol    | ETH                          | ETH                          | SepoliaETH                             |
| Block explorer URL | https://arbiscan.io          | https://nova.arbiscan.io/    | https://sepolia-explorer.arbitrum.io/  |

### Step 3: Initiate the deposit

> To bridge your ETH or ERC-20 tokens to a different chain, start by visiting [bridge.arbitrum.io](https://bridge.arbitrum.io/). Login to the bridge with your wallet and make sure you are connected to the source network (from where you want to deposit your assets) at the top of the page. Then, select the destination network (where you want your assets to go), e.g., Arbitrum One or Arbitrum Nova.

Reviewer: ```suggestion
To bridge your ETH or ERC-20 tokens to a different chain, start by visiting [bridge.arbitrum.io](https://bridge.arbitrum.io/). Log in to the bridge with your wallet and make sure you are connected to the source network (from where you want to deposit your assets) at the top of the page. Then, select the destination network (where you want your assets to go), e.g., Arbitrum One or Arbitrum Nova.
```

⚠️ Note that testnets like Ethereum Sepolia or Ethereum Goerli only appear if you are connected to the appropriate child testnet network (Arbitrum Sepolia or Arbitrum Goerli).

![Select the token to withdraw](assets/getting_started_users_5.png)

Select the token you want to bridge in the token drop-down menu. You can also enable/disable the token lists by clicking `Manage token lists` button on the bottom right corner of the drop-down menu. Enter the amount of ETH or ERC-20 tokens you want to bridge over in the **from** box and then press `Move funds`. Follow the prompts on your web3 wallet.

:::info Ensure sufficient ETH balance

Please make sure you leave enough ETH on your wallet to pay for the transaction, otherwise there will be no web3 wallet popup.

:::

![Enter the amount of token to withdraw](assets/getting_started_users_6.png)

A countdown will pop up stating that you’ll get your funds in 7-8 days.

You can check the status of your withdrawal by clicking on your profile on the top right and opening the `Transactions` tab, and claim it there when it’s ready.

![See the transaction history](assets/getting_started_users_7.png)

> Once the countdown is done, switch to the destination network on your wallet and press the `Claim` button that has now turned blue to receive your funds!

Reviewer: ```suggestion
Once the countdown is done, switch to the destination network on your wallet and press the `Claim` button, that has now turned blue, to receive your funds!
```

> ## What's Next?

Reviewer: ```suggestion
## What's next?
```
